src/plugin.js

```diff
@@ -10,22 +10,33 @@ import OrbitOrientationContols from './orbit-orientation-controls.js';
 import * as utils from './utils';
 import CanvasPlayerControls from './canvas-player-controls';
 import OmnitoneController from './omnitone-controller';
+import { DeviceOrientationControls } from '../vendor/three/DeviceOrientationControls.js';
+import imgUx from './img';
 
-// import controls so they get regisetered with videojs
-import './cardboard-button';
+// WebXR related imports
+import WebXRPolyfill from 'webxr-polyfill';
+import {VRButton} from '../vendor/three/VRButton.js';
+import {XRControllerModelFactory} from '../node_modules/three/examples/jsm/webxr/XRControllerModelFactory';
+import {BoxLineGeometry} from '../node_modules/three/examples/jsm/geometries/BoxLineGeometry';
```

Comment:
These can be imported from three directly I think?

```suggestion
import {XRControllerModelFactory} from 'three/examples/jsm/webxr/XRControllerModelFactory';
import {BoxLineGeometry} from 'three/examples/jsm/geometries/BoxLineGeometry';
```